custom_dc/troubleshooting.md

```diff
@@ -38,15 +38,16 @@ Step 7/62 : COPY mixer/go.mod mixer/go.sum ./
 COPY failed: file not found in build context or excluded by .dockerignore: stat mixer/go.mod: file does not exist
 ```
 
-You need to download additional git modules. See [One-time setup: download build dependencies](build_repo.md#download_deps).
+You need to download additional git modules. See [One-time setup: download build dependencies](/custom_dc/build_repo.html#download_deps).
```

Comment:
Is this the right link? I don't see a mention of download dependencies on the build_repo.html page.

Reply:
Oh crap, that is one I actually changed (and forgot to update -- which is why I think we need a link checker :-)) Will fix.